app/models/track.py

```diff
@@ -9,19 +9,21 @@ class Track(db.Model):
     description = db.Column(db.Text)
     track_image_url = db.Column(db.Text)
     color = db.Column(db.String, nullable=False)
+    font_color = db.Column(db.String)
```

Comment:
@niranjan94 how can we provide font color in the api without saving in database ?